.github/workflows/docker-image.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [ master, dev-build ]
```

Comment:
i suggest removing master from this and add release_docker here so random commits on master don't overwrite the latest tagged already released docker images

Reply:
removed master and added release_docker.